### PR TITLE
Add LAME 3.100 to the Mac build. #32

### PR DESCRIPTION
**Pull request by rryan**

### CODE REVIEW

**Comment by rryan** [comment]
libmp3lame built successfully: https://builds.renegadetech.mixxx.org/job/v2.2/job/buildserver-2.2.x-macosx-prs/architecture=amd64,only_buildserver_capable_nodes=buildserver,platform=macosx/ws/environment/2.2-PR37-j00011-4c7fcce-osx10.7-x86_64-release/lib/

**Comment by rryan** [comment]
@sblaisot?

File: scripts/macosx/build_lame.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+for ARCH in ${MIXXX_ARCHS[@]}
+do
+  mkdir -p $LAME_VERSION-$ARCH
+  tar -zxf $DEPENDENCIES/$LAME_ARCHIVE -C $LAME_VERSION-$ARCH --strip-components 1
+  cd $LAME_VERSION-$ARCH
+  source $PROGDIR/environment.sh $ARCH
+  # Drop non-existent symbol.
+  # https://hydrogenaud.io/index.php/topic,114777.0.html
+  sed -i'' -e '/lame_init_old/d' include/libmp3lame.sym
```
Comment (gramanas) [inline scripts/macosx/build_lame.sh:32]: Can't this be just `sed -i '/lame_init_old/d' include/libmp3lame.sym`

Comment (rryan) [reply scripts/macosx/build_lame.sh:32]: Annoyingly, macOS's sed takes a non-optional extension parameter:
https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/sed.1.html

**Comment by rryan** [comment]
Going to go ahead with this. We can test that it works when we switch the golden environment.

**Pull request merged**